src/LibKubernetesGenerator/templates/Client.cs.template

```diff
@@ -0,0 +1,154 @@
+// <auto-generated>
+// Code generated by https://github.com/kubernetes-client/csharp/tree/master/src/LibKubernetesGenerator
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+namespace k8s.ClientSets;
+
+
+/// <summary>
+/// </summary>
+public partial class {{name}}Client : ResourceClient
+{
+       public {{name}}Client(IKubernetes kubernetes) : base(kubernetes)
+       {
+       }
+
+   {{for api in apis }}
+    {{if IfReturnType api.operation "void"}}
+    private async Task<HttpOperationResponse> {{name}}Client_{{GetMethodName api.operation "WithHttpMessagesAsync"}}(
```

Comment:
is it possible to delegate here to older api
or make older api calling newer api?

better to have single entrypoint